### DISCUSSION TITLE
lotus wallet without syncing the network

### DISCUSSION THREAD

**Discussion by the42** [filecoin-project/lotus#5646] [General Q&A]
I only have 8GB of Ram. I tried to participate in lotus daemon but after syncing and getting at real work, my system hangs. I guess it is swapping to death. No wonder, the minimum system requirements are 32Gb. I am fine with that.

I created a wallet like `lotus wallet new`. I plan to receive FILs from a market on that address. My issue is:

Will I be able to send FILs with lotus when running lotus in this "mode" : `lotus daemon --bootstrap=false`

If not, my Filecoins are locked as I will never be able to send them. What options do I have then?

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
> What options do I have then?

you could use a web wallet and import the private key

wallet.filfox.info or something like that, not sure right now if they allow importing

https://github.com/filecoin-project/community/blob/master/projects/wallet/sovereign.md looks nice. haven't tried it

https://docs.filecoin.io/reference/ might be worth a look to find something that helps you out

sorry, cannot really answer the --bootstrap question

**Reply by the42** [reply]
Thank you about sovereign. As I understand it it is currently Mac only. I asked my question there too: https://github.com/filecoin-project/community/discussions/84
